birdhouse/config/canarie-api/docker_configuration.py.template

```diff
@@ -234,14 +242,351 @@ SERVICES = {
+            'version': "${GEOSERVER_VERSION}",
+            'institution': 'Ouranos',
+            'releaseTime': get_release_time_from_repo_tag("docker", "${GEOSERVER_DOCKER}", "${GEOSERVER_TAGGED}"),
+            'researchSubject': 'Geospatial',
+            'supportEmail': '${SUPPORT_EMAIL}',
+            'category': 'Data Catalog',
+            'tags': ['Data', 'Geospatial', 'Catalog', 'OGC', 'WFS', 'WMS', 'WPS']
+        },
+        'stats': {
+            'method': '.*',
+            'route': "/geoserver/.*"
+        },
+        'redirect': {
+            'doc': 'https://docs.geoserver.org/',
+            'releasenotes': 'https://geoserver.org/release/${GEOSERVER_VERSION}/',
+            'support': 'https://github.com/kartoza/docker-geoserver/issues',
+            'source': 'https://github.com/kartoza/docker-geoserver',
+            'tryme': 'https://${PAVICS_FQDN_PUBLIC}/geoserver/',
+            'licence': 'https://github.com/geoserver/geoserver/blob/${GEOSERVER_VERSION}/LICENSE.txt',
+            'provenance': 'https://github.com/kartoza/docker-geoserver'
+        },
+        "monitoring": {
+            "GeoServer": {
+                'request': {
+                    'url': 'https://${PAVICS_FQDN_PUBLIC}/geoserver/web/'
+                }
+            }
+        }
+    },
+    'Phoenix': {
+        'info': {
+            'name': 'Phoenix',
+            'synopsis': 'Legacy authentication. See Magpie/Twitcher instead.',
+            'version': "${PHOENIX_VERSION}",
+            'institution': 'Ouranos',
+            'releaseTime': get_release_time_from_repo_tag("docker", "${PHOENIX_DOCKER}", "${PHOENIX_VERSION}"),
+            'researchSubject': 'Authentication',
+            'supportEmail': '${SUPPORT_EMAIL}',
+            'category': 'Authentication',
+            'tags': ['Authentication', 'Legacy']
+        },
+        'stats': {
+            'method': '.*',
+            'route': '${TWITCHER_PROTECTED_PATH}/geoserver/web/.*'  # FIXME: original value doesn't make sense
+        },
+        'redirect': {
+            'doc': 'http://pyramid-phoenix.readthedocs.io/en/latest/index.html',
+            'releasenotes': 'https://github.com/ouranosinc/pyramid-phoenix/CHANGES.rst',
+            'support': 'https://github.com/ouranosinc/pyramid-phoenix/issues',
+            'source': 'https://github.com/ouranosinc/pyramid-phoenix',
+            'tryme': 'https://${PAVICS_FQDN_PUBLIC}/geoserver/web/',  # FIXME: original value doesn't make sense
+            'licence': 'https://github.com/ouranosinc/pyramid-phoenix/blob/master/LICENSE.txt',
+            'provenance': 'https://ouranosinc.github.io/pavics-sdi/provenance/index.html'
+        },
+        "monitoring": {
+            "Phoenix": {
+                'request': {
+                    'url': 'https://${PAVICS_FQDN_PUBLIC}/geoserver/web/'  # FIXME: original value doesn't make sense
```

Comment:
Yeah that's wierd geoserver monitoring url for phoenix in the original value.

Reply:
Should we consider `phoenix` as another once to be removed in scope of #214 ?

Reply:
Oh maybe, I am not aware anyone on Ouranos side use it, you probably know on CRIM side, but we probably should ask PCIC, just in case.

Reply:
Ok, I'll remove the config since its not monitoring it anyway. The actual component removal can be considered later.

Reply:
Well... I cannot remove just the monitoring part as it fails schema validation. I'll simply adjust the URL to match where it actually is accessible, and the config can be removed entirely when the component is removed.